btc-withdrawal/src/main/kotlin/com/d3/btc/withdrawal/init/BtcWithdrawalInitialization.kt

```diff
@@ -120,39 +113,15 @@ class BtcWithdrawalInitialization(
                 )
             }
         }
-        // Handle 'broadcast' events
-        getSetDetailCommands(block).filter { command -> isBroadcast(command) }
-            .forEach { command ->
-                broadcastTransactionHandler.handleBroadcastCommand(command.setAccountDetail)
-            }
-        // Handle 'create new transaction' events
-        getSetDetailCommands(block).filter { command -> isNewTransactionCreated(command) }
-            .forEach { command ->
-                newTransactionCreatedHandler.handleCreateTransactionCommand(command.setAccountDetail)
-            }
-        // Handle signature appearance commands
-        getSetDetailCommands(block).filter { command -> isNewWithdrawalSignature(command) }
-            .forEach { command ->
-                newSignatureEventHandler.handleNewSignatureCommand(
-                    command.setAccountDetail
-                ) { transferWallet.saveToFile(File(btcWithdrawalConfig.btcTransfersWalletPath)) }
-            }
-        // Handle 'set new consensus' events
-        getSetDetailCommands(block).filter { command -> isNewConsensus(command) }
-            .forEach { command ->
-                newConsensusDataHandler.handleNewConsensusCommand(command.setAccountDetail)
+        // Handle other commands
+        getSetDetailCommands(block).map { it.setAccountDetail }
+            .forEach { setAccountDetailCommand ->
+                accountDetailHandlers.forEach { handler ->
+                    if (handler.filter(setAccountDetailCommand)) {
```

Comment:
maybe it is better to make `filter` a part of `handle`? just not to forget to `filter` somewhere else